src/components/TodoMain/TodoMain.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { TodoItem } from '../TodoItem';
+
+type Prop = {
+  todos: Todo[];
+  toggleTodo: (todo: Todo) => void | Promise<void>;
+  onDeleted: (id: number) => void;
+  deletingTodoId: number | null;
+  deletingAllTodo: number[] | null;
+  loadingTodoId: number | null;
+  loadingAllTodo: number[] | null;
+  onEditSubmit: (
+    id: number,
+    oldTitle: string,
+    newTitle: string,
+  ) => void | Promise<void>;
+};
+
+export const TodoMain: React.FC<Prop> = ({
```

Comment:
What does Main mean in the component name? It's a list - rename component accordingly